docs/sync/gitlab_ci.md

```diff
@@ -188,6 +195,7 @@ A GitLab token with API access is required to use the API (e.g., to post notes/c
 project, or group access token. The account used to create the token will be the one that appears to post the
 notes/comments on the MR. Therefore, it might be worth looking into using a bot account, which is available for
 project and group access tokens. See the [GitLab Token Overview][gitlab_tokens] documentation for more info.
+The token needs `api` scope. Project or Group access tokens should specify a role _other than_ `Guest`.
```

Comment:
```suggestion
The token needs the `api` scope. Project or Group access tokens should specify a role _other than_ `Guest`.
```

Reply:
The change was applied...but in _both_ places it occurs.